db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180410144440) do
+ActiveRecord::Schema.define(version: 20180410060459) do
```

Comment:
Why do you need to update the schema version? since there was no modification on db migration.